tf/examples/EMI-RNN/00_emi_lstm_example_automode_false.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using EMI-RNN on the HAR Dataset\n",
+    "\n",
+    "This is a very simple example of how the existing EMI-RNN implementation can be used on the HAR dataset. We illustrate how to train a model that predicts on 48 step sequence in place of the 128 length baselines while attempting to predict early. For more advanced use cases which involves more sophisticated computation graphs or loss functions, please refer to the doc strings provided with the released code.\n",
```

Comment:
The last few sentences now are redundant as you've implemented `automode` and have enabled this behaviour. Since this notebook is to serve as an example to custom loss functions and optimizers, it would be nice if you could add that information here.